Editor/PostBuild.cs

```diff
@@ -1,130 +1,229 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
 using System.Linq;
-using System.Threading;
 using System.Threading.Tasks;
 using BugSplatDotNetStandard;
 using UnityEditor;
 using UnityEditor.Callbacks;
 using UnityEngine;
 using BugSplatUnity.Runtime.Client;
+#if UNITY_IOS
+using UnityEditor.iOS.Xcode;
+#endif
 
 public class BuildPostprocessors
 {
-    static string _platform;
-
-    /// <summary>
-    /// Upload Asset/Plugin symbol files to BugSplat. 
-    /// We don't upload Unity symbol files because the build output only contains public symbol information.
-    /// BugSplat is configured to use the Unity symbol server which has private symbols containing file, function, and line information.
-    /// </summary>
-    [PostProcessBuild(1)]
-    public static async Task OnPostprocessBuild(BuildTarget target, string pathToBuiltProject)
-    {
-        switch (target)
-        {
-            case BuildTarget.StandaloneWindows64: _platform = "x86_64"; break;
-            case BuildTarget.StandaloneWindows: _platform = "x86"; break;
-            default: return;
-        }
-
-        var projectDir = Path.GetDirectoryName(Application.dataPath);
-        var pluginsDir = Path.Combine(Path.Combine(projectDir, "Assets", "Plugins"), _platform);
-
-        if (!Directory.Exists(pluginsDir))
-        {
-            UnityEngine.Debug.Log("Plugins directory doesn't exist, skipping symbol uploads...");
-            return;
-        }
-
-        await UploadSymbolFiles(pluginsDir);
-    }
-
-    static async Task UploadSymbolFiles(string pluginsDir)
-    {
-        var options = GetBugSplatOptions();
-
-        if (options == null)
```

Comment:
Why did this get removed?

Reply:
The null check is now present in the top-most method.
`public static async Task OnPostprocessBuild(BuildTarget target, string pathToBuiltProject)`

Reply:
👍 must've missed that, thanks!